tests/layer_and_mask_information_section.rs

```diff
@@ -249,3 +249,23 @@ fn group_by_name<'a>(psd: &'a Psd, name: &str) -> &'a PsdGroup {
         .unwrap()
         .1
 }
+
+/// cargo test --test layer_and_mask_information_section layer_larger_than_psd -- --exact
+#[test]
+fn layer_larger_than_psd() {
+    let psd = include_bytes!("./fixtures/layer-larger.psd");
```

Comment:
Mind documenting these new fixtures in here https://github.com/namse/psd/blob/master/tests/fixtures/README.md